src/_reactComponents/PanelHeaderComponents/Button.jsx

```diff
@@ -1,54 +1,49 @@
-import React, { useState } from 'react';
```

Comment:
What task are you completing in this file? Creates weird right margin for buttons used in menus.
<img width="242" alt="Screen Shot 2022-05-15 at 1 32 06 PM" src="https://user-images.githubusercontent.com/68914928/168488395-ea9e95bb-65fc-45d5-80ab-738033fb5858.png">